api/envoy/extensions/rbac/audit_loggers/stream/v3/stream.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+
+package envoy.extensions.rbac.audit_loggers.stream.v3;
+
+import "udpa/annotations/status.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.rbac.audit_loggers.stream.v3";
+option java_outer_classname = "StreamProto";
+option java_multiple_files = true;
+option go_package = "github.com/envoyproxy/go-control-plane/envoy/extensions/rbac/audit_loggers/stream/v3;streamv3";
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: Standard Streams RBAC Audit Loggers]
+// [#not-implemented-hide:]
+
+// Custom configuration for the RBAC audit logger that writes log entries
+// directly to the operating system's standard output.
+// The logger outputs in JSON format and is currently not configurable.
+message StdoutAuditLog {
```

Comment:
stdout is only one type of log sink, that may not be usable by everyone. Please add something similar to the request access log configuration https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/accesslog/v3/accesslog.proto#config-accesslog-v3-accesslog which allows different implementations of log sinks (i.e. gRPC).

You can exclude the `filter` field, since there are no defined filtering semantics for RBAC log yet (although I can see someone only logging denied requests).